barnes.gemspec

```diff
@@ -10,6 +10,7 @@ Gem::Specification.new do |s|
   s.email     = 'agwozdziewycz@salesforce.com'
```

Comment:
Perhaps we should also change this?

Reply:
Email is a bit of a formality IMHO. We can add multiple authors via `s.authors` but also it's not really used for anything. There's a command in rubygems that grants permission.

Reply:
I have no idea how to Gem, so I just put some stuff in there.